examples/llama.py

```diff
@@ -311,7 +314,7 @@ def build(model_path, tokenizer_path, model_gen="1", model_size="7B", quantize=F
     else:
       weights = load(str(model_path))
     if "model.embed_tokens.weight" in weights:
-      weights = convert_from_huggingface(weights, model, model_args["n_heads"], model_args["n_kv_heads"])
+      weights = convert_from_huggingface(weights, model, model_args["n_heads"], model_args.get("n_kv_heads"))
```

Comment:
Can't you do a get here with a default value of `n_heads`? saves a couple lines.

Reply:
Done. Though I'd personally say LOC is less important for code on `/examples` - I think example code can be ungolfed as much as possible and even tolerate some comment noise.